src/playback-watcher.js

```diff
@@ -44,14 +44,15 @@ export default class PlaybackWatcher {
     }
     this.logger_('initialize');
 
+    let canPlayHandler = () => this.monitorCurrentTime_();
     let waitingHandler = () => this.techWaiting_();
     let cancelTimerHandler = () => this.cancelTimer_();
     let fixesBadSeeksHandler = () => this.fixesBadSeeks_();
 
     this.tech_.on('seekablechanged', fixesBadSeeksHandler);
     this.tech_.on('waiting', waitingHandler);
     this.tech_.on(timerCancelEvents, cancelTimerHandler);
-    this.monitorCurrentTime_();
+    this.tech_.on('canplay', canPlayHandler);
```

Comment:
you should `off` the handler in the dispose method below like the other handlers